File: app/src/main/java/com/example/lumuttest/data/remote/ApiService.java
package com.example.lumuttest.data.remote;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Path;

public interface ApiService {
    @GET("/todos")
    Call<List<Todo>> getTodos();

    @GET("/todos/{id}")
    Call<Todo> getTodoById(@Path("id") int id);
}

File: app/src/main/java/com/example/lumuttest/data/remote/ApiService.kt
package com.example.lumuttest.data.remote

import retrofit2.http.GET
import retrofit2.http.Path

interface ApiService {
    @GET("/todos")
    suspend fun getTodos(): List<Todo>

    @GET("/todos/{id}")
    suspend fun getTodoById(@Path("id") id: Int): Todo
}
